src/zone.ts

```diff
@@ -0,0 +1,80 @@
+import { parseZoneFile, makeZoneFile } from "zone-file";
+import * as fs from 'fs';
+
+export function getDefaultAddress(coin: string, text: any | string): { [key: string]: string } | null {
+    let addresses = getAddresses(coin, text);
+    return (addresses == null ? null : addresses[0]);
+}
+
+export function getAddresses(coin: string, text: any | string): { [key: string]: string }[] | null {
+    // what type to use for text?
+    let zoneFileJson = (typeof text === 'string' ? parseZoneFile(text) : text);
```

Comment:
if the `text` argument is the zone file, why not name it `zonefile`?

Reply:
yup changed